book.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'item'
+
+class Book < Item
+  attr_accessor :cover_state
+  attr_reader :publisher
+
```

Comment:
I believe you we'll ever need to access the `cover_state` directly. Therefore, I'd suggest we set the cover_state under `attr_reader` and remove the test in the `book_spec.rb` file.